src/leiningen/new/db.clj

```diff
@@ -89,6 +94,25 @@
                                         ['com.google.guava/guava "25.1-jre"]
                                         ['io.rkn/conformity "0.5.1"]])))])
 
+(defn add-crux [[assets options]]
+  [(into assets crux-files)
+   (-> options
+       (assoc
+         :crux true
+         :db-connection true
+         :db-docs ((:selmer-renderer options) (slurp-resource "db/docs/crux_instructions.md") options)
+         :database-profile-dev (str :crux-config "\n"
+                                    (with-out-str (clojure.pprint/pprint
```

Comment:
The formatting of the config is a little funny, maybe there's a nicer way of doing this? Currently comes out like this:

```clojure
 :crux-config
{:crux/index-store
 {:kv-store
  {:crux/module crux.rocksdb/->kv-store, :db-dir "data/indices"}},
 :crux/document-store
 {:kv-store
  {:crux/module crux.rocksdb/->kv-store, :db-dir "data/docs"}},
 :crux/tx-log
 {:kv-store
  {:crux/module crux.rocksdb/->kv-store, :db-dir "data/transactions"}}}
}
```